com.revolutionarygamesstudio.ThriveLauncher.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.revolutionarygamesstudio.ThriveLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: thrivelauncher
+separate-locales: false
+rename-icon: thrive-launcher
+finish-args:
+  - --share=network
+  - --device=dri
+  - --socket=x11
```

Comment:
x11 always want `--share=ipc`

Reply:
Why?

Reply:
because. it always needs. without it it is slower.